### ISSUE TITLE
ASR的结果怎么带上时间戳呢？

### ISSUE THREAD

**Issue by jinec** [QwenLM/Qwen2-Audio#136]
我现在在做关于 会议助手的 项目，需要带上时间戳，后面方便进行 说话人确认。我现在的代码是仿照 样例上的，如下所示：
def transcribe_chunks(chunks, sample_rate, data_prompt, batch_size=8):
    results = []
    for i in range(0, len(chunks), batch_size):
        batch = chunks[i:i + batch_size]
        audios = [chunk.squeeze().numpy() for chunk in batch]
        prompts = ["<|audio_bos|><|AUDIO|><|audio_eos|>" + data_prompt for _ in batch]
        inputs = processor(text=prompts, audios=audios, sampling_rate=sample_rate, return_tensors="pt", padding=True)
        inputs = {k: v.to(device) for k, v in inputs.items()}
        with torch.no_grad():
            outputs = model.generate(**inputs, max_new_tokens=512, do_sample=False)
        output_ids = outputs[:, inputs['input_ids'].shape[1]:]
        batch_result = processor.batch_decode(output_ids, skip_special_tokens=True, clean_up_tokenization_spaces=False)
        results.extend(batch_result)
    return results

# 初始化模型和处理器
model_id = "/workspace/project/models/Qwen/Qwen2-Audio-7B"
processor = AutoProcessor.from_pretrained(model_id)
processor.tokenizer.padding_side = 'left'
model = Qwen2AudioForConditionalGeneration.from_pretrained(
    model_id,
    device_map='cuda',
    torch_dtype='auto',
    trust_remote_code=True
).eval()
